From b9e9348d381ea2e2a8782e753a686cf57c26ec60 Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Tue, 22 Oct 2019 18:19:22 +0200
Subject: [PATCH] reset: fix reset_control_get_exclusive kerneldoc comment
Git-commit: b9e9348d381ea2e2a8782e753a686cf57c26ec60
Patch-mainline: v5.4-rc7
References: bsc#1051510

Add missing parentheses to correctly hyperlink the reference to
reset_control_get_shared().

Fixes: 0b52297f2288 ("reset: Add support for shared reset controls")
Reviewed-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 include/linux/reset.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/reset.h
+++ b/include/linux/reset.h
@@ -101,7 +101,7 @@ static inline struct reset_control *__de
  * If this function is called more then once for the same reset_control it will
  * return -EBUSY.
  *
- * See reset_control_get_shared for details on shared references to
+ * See reset_control_get_shared() for details on shared references to
  * reset-controls.
  *
  * Use of id names is optional.
